### PR TITLE
feat(route): add 5EPLAY News

### LINKED ISSUE #12474: 5EPLAY RSS订阅源
### 类型

新媒体

### 网站地址

https://www.5eplay.com/

### 网站描述

这是一个新媒体网站，每日提供最新的电竞新闻

### 需要生成什么内容？

提取网站每日最新文章，点进去能够查看文章图片和正文（就像IT之家 RSS 一样的效果）

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by Dlouxgit**
## 该 PR 相关 Issue / Involved Issue

Close #12474 

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/5eplay/article
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/5eplay/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    let acw_sc__v2 = '';
+    // get arg1
+    await ctx.cache.tryGet(
+        articleUrl,
+        async () => {
+            // Zlib Z_BUF_ERROR, should close decompress
+            const detailResponse = await got(articleUrl, {
+                decompress: false,
+            });
```
Comment (TonyRL) [inline lib/v2/5eplay/index.js:27]: Please keep your coding style consistent.
https://github.com/DIYgod/RSSHub/blob/76eef551769350183dea7841f56ef71606bf5ac8/lib/v2/5eplay/index.js#L12-L15
https://github.com/DIYgod/RSSHub/blob/76eef551769350183dea7841f56ef71606bf5ac8/lib/v2/5eplay/index.js#L25-L27
https://github.com/DIYgod/RSSHub/blob/76eef551769350183dea7841f56ef71606bf5ac8/lib/v2/5eplay/index.js#L47-L53

File: lib/v2/5eplay/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            const obj_unzip = zlib.createUnzip({
+                chunkSize: 20 * 1024,
+            });
+            obj_unzip.write(detailResponse.body);
+            const arg1 = await new Promise((resolve) => {
+                obj_unzip.on('data', (data) => {
+                    resolve(data.toString().match(/var arg1='(.*?)';/)[1]);
+                });
+            });
+            acw_sc__v2 = getAcwScV2ByArg1(arg1);
```
Comment (TonyRL) [inline lib/v2/5eplay/index.js:38]: I think it's possible to do it with loop
```js
            for await (const data of obj_unzip) {
                const strData = data.toString();
                const matches = strData.match(/var arg1='(.*?)';/);
                if (matches) {
                    acw_sc__v2 = getAcwScV2ByArg1(matches[1]);
                    break;
                }
            }
```

File: lib/v2/5eplay/index.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    const { data: response } = await got({
+        method: 'get',
+        url: apiUrl,
+    });
+
+    const expiredTime = 25 * 60 * 1000;
+
+    let acw_sc__v2 = '';
+    // get arg1
+    await ctx.cache.tryGet(
```
Comment (TonyRL) [inline lib/v2/5eplay/index.js:21]: This `tryGet` function has no return value which means nothing will be stored in the cache.
Beside that, `acw_sc__v2` will be `''` when cache-hit.
docs: https://docs.rsshub.app/joinus/use-cache.html

File: lib/v2/5eplay/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    const rootUrl = 'https://csgo.5eplay.com/';
+    const apiUrl = `${rootUrl}api/article?page=1&type_id=0&time=0&order_by=0`;
+    const articleUrl = `${rootUrl}article`;
+
+    const { data: response } = await got({
+        method: 'get',
+        url: apiUrl,
+    });
+
+    const expiredTime = 25 * 60 * 1000;
```
Comment (TonyRL) [inline lib/v2/5eplay/index.js:17]: Please use `config.cache.routeExpire` instead. Note that the unit for all cache-related function is second.
```js
const config = require('@/config').value;
```

File: lib/v2/5eplay/index.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        false
+    );
+
+    const items = await Promise.all(
+        response.data.list.map((item) =>
+            ctx.cache.tryGet(item.jump_link, async () => {
+                if (!acw_sc__v2) {
+                    return {
+                        title: item.title,
+                        description: item.title + (item.images?.[0] ? `<img src="${item.images[0]}" referrerpolicy="no-referrer">` : ''),
```
Comment (TonyRL) [inline lib/v2/5eplay/index.js:59]:
```suggestion
                        description: item.title + (item.images?.[0] ? `<img src="${item.images[0]}"` : ''),
```
No need to add referer policy. This is handled by RSSHub
https://github.com/DIYgod/RSSHub/blob/db56ef5fcade70863fad3062d63046b2d7b83470/lib/middleware/parameter.js#L137-L139

File: lib/v2/5eplay/index.js (hunk 2 above)
Comment (Dlouxgit) [reply lib/v2/5eplay/index.js]: Awesome, I didn't know it was possible to write code like this and it works

**Pull request merged**